scripts/exregional_run_analysis.sh

```diff
@@ -473,7 +501,7 @@ else
   esac
 fi
 
-if [[ ${gsi_type} == "OBSERVER" || ${ob_type} == "conv" ]]; then
+if [[ ${gsi_type} == "OBSERVER" || ${anav_type} == "conv" || ${anav_type} == "all" ]]; then
```

Comment:
Using "all" as analysis variable is too general. As this application doing all the thing and cannot be extended?